xbmc/cores/DataCacheCore.h

```diff
@@ -163,4 +175,7 @@ class CDataCacheCore
     int64_t m_timeMax;
     int64_t m_timeMin;
   } m_timeInfo = {};
+
+  std::vector<EDL::Cut> m_cutList;
```

Comment:
The purpose of ProcessInfo is to hold information about the configuration of VP, not about the content. Adding cut list violates the original design.